src/NzbDrone.Core/Download/RedownloadFailedDownloadService.cs

```diff
@@ -58,7 +61,7 @@ public void HandleAsync(DownloadFailedEvent message)
 
             _logger.Debug("Failed download contains multiple episodes, probably a double episode, searching again");
 
-            _commandExecutor.PublishCommandAsync(new EpisodeSearchCommand(message.EpisodeIds));
+            _commandService.PublishCommand(new EpisodeSearchCommand(message.EpisodeIds));
```

Comment:
Ewwww manager? I agree service isn't good here, I think Manager actually fits in this case though.